test/gtest/configure.m4

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2019.  ALL RIGHTS RESERVED.
+# Copyright (C) Huawei Technologies Co., Ltd. 2020.  All rights reserved.
+#
+# See file LICENSE for terms.
+#
+
+AC_LANG_PUSH([C++])
+
+CHECK_COMPILER_FLAG([-fno-tree-vectorize], [-fno-tree-vectorize],
```

Comment:
why we need to check if compiler supports --diag_suppress 236 and fno-tree-vectoeize? Does not look needed?

Reply:
I traced this to https://github.com/openucx/ucx/commit/6621f7019f318df8eda4020828497578e94849dd - where it was added in UCX. probably to fix some compiler-specific problem. I can remove it - but then we will likely run into the same problem. What's your opinion? @vspetrov